openapi/paths/alias/setTLD.json

```diff
@@ -0,0 +1,79 @@
+{
+  "post": {
+    "summary": "Set Custom Alias TLD",
+    "description": "Create a [Top Level Domain (TLD)](https://github.com/signum-network/SIPs/blob/master/SIP/sip-48.md) for aliases. A TLD is like a namespace for Aliases, such it is possible to have same Aliases in different TLDs. Note, that acquisition of a custom TLD charges 100K SIGNA",
```

Comment:
Mabye saying that this is to pay as amount (100k) - or can we fix the value of the amount field with 100k ?

Reply:
I removed it